README.md

```diff
@@ -88,6 +92,39 @@ However, if the same message content is posted with `isJSONData` set to `false`,
 }
 ```
 
+Similar to `isJSONData`, if `isBinaryData` was set to `true` during trigger creation, an attempt will be made to encode values from binary to Base64. If successfully encoded, the resultant `value` in the trigger payload will be encoded as a Base64 string.
+
+For example, if a message of `Some data` is posted with `isBinaryData` set to `true`, the trigger payload might look something like this:
```

Comment:
Not sure this example is accurate - "some data" is not binary.. maybe give an example with buffer.